mitodl/open-discussions
# Add Program and UserList to Django admin

#### Pre-Flight checklist


#### What are the relevant tickets?
Indirectly related to #2101, makes it easier to view/edit Program and UserList objects 

#### What's this PR do?
Adds admin pages for Program and UserList objects; fixes the `limit_choices_to` value in those models.

#### How should this be manually tested?
Go to Django admin and you should be able to create/edit/view Programs and UserLists.
